llvm/test/CodeGen/X86/isel-brcond-icmp.ll

```diff
@@ -0,0 +1,1508 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s                               -mtriple=x86_64-apple-darwin10 -verify-machineinstrs | FileCheck %s --check-prefixes=SDAG-X64
+; RUN: llc < %s -fast-isel -fast-isel-abort=1 -mtriple=x86_64-apple-darwin10 -verify-machineinstrs | FileCheck %s --check-prefixes=FASTISEL-X64
```

Comment:
Some of these look like the X86/X64 variants can share a common prefix

Reply:
Yes, the tests with a single argument are the same between X86/X64. I've recently thought that in many cases FastISel and SelectionDAG produce the same output, maybe we can try to unify them under something like `DAG-X86`, `DAG-X64` as they DAG compatible. Or another prefix to put them into the same basket if any.

Reply:
I have no objection if we want to use X86/X64 prefixes for DAG+FASTISEL - and just have GISEL as the special case, especially as we're aiming to remove the diffs entirely. 

Some of the GISEL diffs are just different LBB tags for some reason, which we can hopefully address pretty easily....

Reply:
Makes sense, I made changes to the tests to share more common prefixes.